composer.json

```diff
@@ -34,6 +34,7 @@
     "ext-json": "*",
     "ext-openssl": "*",
     "ext-zip": "*",
+    "christiaanbye/polyfill-each": "^1.0",
```

Comment:
Hi @pankaj0882, we are going to drop this

Reply:
Hi @clemente-raposo dropped the polyfill related commit